linkzstd_darwin.go

```diff
@@ -0,0 +1,6 @@
+package gozstd
```

Comment:
could you rename files starting from `link*` to `lib*`, since they are related to `libzstd` library?

Reply:
Done.

Reply:
Since you need gcc to use cgo on Windows, then you can install MSYS2 to run the Makefile.